Gemfile

```diff
@@ -0,0 +1,10 @@
+source "https://rubygems.org"
+
+git_source(:github) { |repo_name| "https://github.com/#{repo_name}" }
+
+# Uncomment one of the below to aid with development:
+# gem "openactive", git: "https://github.com/openactive/models-ruby.git"
+# gem "openactive", path: "#{__dir__}/../models-ruby"
```

Comment:
Remove this from Gemfile. If worried about helping contributors add something to Contributing section of README.